openmc/statepoint.py

```diff
@@ -92,13 +91,8 @@ class StatePoint(object):
     """
 
     def __init__(self, filename):
-        if filename.endswith('.h5'):
-            import h5py
-            self._f = h5py.File(filename, 'r')
-            self._hdf5 = True
-        else:
-            self._f = open(filename, 'rb')
-            self._hdf5 = False
+        import h5py
```

Comment:
I had the same conflicting thoughts, and settled on leaving it where it is so that you don't absolutely need it just to construct input models.